gatsby-config.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @type {import('gatsby').GatsbyConfig}
+ */
+module.exports = {
+  siteMetadata: {
+    title: `alanlauer.com`,
+    siteUrl: `https://www.yourdomain.tld`
```

Comment:
this one may need to set before deploying works properly to github pages. i believe the url will look like this if we keep the repo name (not an ideal name tho i'd say due to the .com which may affect the url parsing?)

anywho, i think it will look like this: 
`alanlauer.github.io/alanlauer.com`
or (assuming they sanitize): 
`alanlauer.github.io/alanlauer_com`